source/docs/software/hardware-apis/sensors/gyros-software.rst

```diff
@@ -125,6 +152,13 @@ The Pigeon should use the :code:`WPI_PigeonIMU` class.  The Pigeon can either be
         TalonSRX talon{0}; // TalonSRX is on CAN Bus with device ID 0
         WPI_PigeonIMU gyro{talon}; // Pigeon uses the talon created above
 
+    .. code-block:: python
+
+        gyro = WPI_PigeonIMU(0); # Pigeon is on CAN Bus with device ID 0
```

Comment:
Should add `import ctre` and do `ctre.WPI_PigeonIMU`, etc.

Reply:
still working this out and getting the api docs figured out.  Will update when that is working.